packages/mermaid/src/docs/.vitepress/components/TopBar.vue

```diff
@@ -4,14 +4,22 @@
   >
     <p class="flex-grow text-center tracking-wide text-text">
       <a
-        href="https://www.mermaidchart.com/app/user/billing/checkout?coupon=HOLIDAYS2023"
+        href="https://www.mermaidchart.com/app/user/billing/checkout"
         target="_blank"
-        class="unstyled text-white plausible-event-name=bannerClick"
+        class="unstyled flex-grow tracking-wide"
```

Comment:
Why is `plausible-event-name` being removed?

Reply:
Inadvertent removal.  Fixed, thanks.